src/main/java/net/tiagofar78/prisonescape/game/Settings.java

```diff
@@ -1,17 +1,20 @@
 package net.tiagofar78.prisonescape.game;
 
 public class Settings {
-	
+
+	private static final double PRISIONERS_RATIO = 0.64;
+	private static final double GUARDS_RATIO = 0.36;
```

Comment:
Move it from here and move it to config manager as a configurable value not as a constant.